anyway/flask_app.py

```diff
@@ -1311,6 +1312,29 @@ def get_dict_file(directory):
         yield name, df
 
 
+@app.route("/markers/polygon/", methods=["GET"])
+def acc_in_area_query():
+    # pol_str will be received in the following format: 'POLYGON(({lon} {lat},{lon} {lat},........,{lonN},{latN}))'
+    # please note that start point and end point must be equal: i.e. lon=lonN, lat=latN
+    pol_str = request.values.get('polygon', "")
+    start_date = utilities.valid_date(request.values.get('start_date', '01-01-2013'))
+    end_date = utilities.valid_date(request.values.get('end_date', '31-12-2017'))
```

Comment:
@erangil you can filter out dates, only if they come from the user, better not to use such default dates

Reply:
Done